crates/miden-objects/src/account/code/mod.rs

```diff
@@ -285,6 +285,36 @@ impl Deserializable for AccountCode {
     }
 }
 
+// PRETTY PRINT
+// ================================================================================================
+
+impl PrettyPrint for AccountCode {
+    fn render(&self) -> vm_core::prettier::Document {
```

Comment:
In addition to to this, it may be good to add something like a `PrintableProcedure` struct. This would implement `PrettyPrint` and would print out a single procedure. The idea is to let user do something like this:
```rust
let printable_procedure = code.get_printable_procedure(procedure_root);
```
The idea here is to allow the users to print procedures individually.